src/backend/marsha/core/tests/test_api_document.py

```diff
@@ -357,50 +356,14 @@ def test_api_document_initiate_upload_instructor(self):
 
         self.assertEqual(response.status_code, 200)
         content = json.loads(response.content)
-        policy = content.pop("policy")
+        self.assertEqual({"url", "fields"}, content.keys())
         self.assertEqual(
-            json.loads(b64decode(policy)),
-            {
-                "expiration": "2018-08-09T00:00:00.000Z",
-                "conditions": [
-                    {"acl": "private"},
-                    {"bucket": "test-marsha-source"},
-                    {
-                        "x-amz-credential": "aws-access-key-id/20180808/eu-west-1/s3/aws4_request"
-                    },
-                    {"x-amz-algorithm": "AWS4-HMAC-SHA256"},
-                    {"x-amz-date": "20180808T000000Z"},
-                    {
-                        "key": (
-                            "27a23f52-3379-46a2-94fa-697b59cfe3c7/document/"
-                            "27a23f52-3379-46a2-94fa-697b59cfe3c7/1533686400.pdf"
-                        )
-                    },
-                    ["content-length-range", 0, 1073741824],
-                ],
-            },
-        )
-        self.assertEqual(
-            content,
-            {
-                "acl": "private",
-                "bucket": "test-marsha-source",
-                "stamp": "1533686400",
-                "key": "{id!s}/document/{id!s}/1533686400.pdf".format(id=document.pk),
-                "max_file_size": 1073741824,
-                "s3_endpoint": "s3.eu-west-1.amazonaws.com",
-                "x_amz_algorithm": "AWS4-HMAC-SHA256",
-                "x_amz_credential": "aws-access-key-id/20180808/eu-west-1/s3/aws4_request",
-                "x_amz_date": "20180808T000000Z",
-                "x_amz_expires": 86400,
-                "x_amz_signature": (
-                    "f5b19f6167f076a92f77bcb30fbd5474bdd749093ba7095eaa8e820830859227"
-                ),
-            },
+            content.get("fields").get("key"),
```

Comment:
Why don't you keep the test to the same level of checks ie checking the whole "content" object?

Reply:
Because all of this is computed by boto3 and not by us anymore so I don't know if it's relevant to test these computed values.

Reply:
I think it is :man_shrugging:

Reply:
Ok but some values are computed with a date and the date is not mock anymore, we can't compute `expiration` and `x-amz-date` conditions

Reply:
IC. Maybe test whatever you can without too much hassle?